src/common/options/global.yaml.in

```diff
@@ -2568,7 +2568,7 @@ options:
   type: str
   level: advanced
   desc: default erasure code profile for new erasure-coded pools
-  default: plugin=jerasure technique=reed_sol_van k=2 m=2
+  default: plugin=isa technique=reed_sol_van k=2 m=2
```

Comment:
I'm afraid the change isn't going to be as simple as `s/jerasure/isa/`. The problem is we still need to test `jerasure` in upstream for the sake of existing clusters. I think the same idiom like with the`Filestore`-to-`BlueStore` transition is needed.

Reply:
I had the same suspicion.